SeleniumPracticalExercise/PageObjects/AddEmployeePage.cs

```diff
@@ -17,11 +17,11 @@ public AddEmployeePage(IWebDriver driver) : base(driver)
         }
 
         /// <summary>
-        /// Enter Firstname & Lastname
-        /// then Click Save
-        /// did stale so we can wait for element to save. 
+        /// Enter First and Last name
         /// </summary>
-        public void SetEnterNames(string firstname, string lastname)
+        /// <param name="firstname"></param>
+        /// <param name="lastname"></param>
+        public void SetNames(string firstname, string lastname)
```

Comment:
I missed this before... 'firstname' should be 'firstName' and so on... camel case for variable names here. Check this everywhere else as well.

Reply:
Also... make sure you use CTRL+R, CTRL+R to rename the variable so that all instances, including those in the XML docs, are renamed.